flow/__main__.py

```diff
@@ -46,6 +49,43 @@ def main_init(args):
         )
 
 
+def main_template(args):
+    """Print out general information about template subcommand."""
+    print("flow template does nothing. Use of the available subcommands [create,].")
+
+
+def main_template_create(args):
+    """Create custom template based on a specified or detected environment."""
+    if args.name is None:
+        env = environment.get_environment()
+        if env is environment.StandardEnvironment:
+            extend_template = "base_script.sh"
+        else:
+            extend_template = env.template
+    else:
+        extend_template = args.name
+
+    project = get_project()
+    os.makedirs(project.fn("templates"), exist_ok=True)
+
+    # grab and render custom template
+    jinja_env = jinja2.Environment(loader=jinja2.PackageLoader("flow"))
+    custom_template_template = jinja_env.get_template("custom.sh")
+    new_template = custom_template_template.render(extend_template=extend_template)
+
+    try:
+        with open(project.fn(os.path.join("templates", "script.sh")), "x") as fh:
+            fh.write(new_template)
+    except OSError as error:
+        if error.errno == errno.EEXIST:
+            logger.error(
+                "Error while trying to create custom template. Delete "
+                "'templates/script.sh' first and rerun command."
```

Comment:
Indicate the cause of the error before recommending the solution.
```suggestion
                "The file 'templates/script.sh' already exists. Delete "
                "'templates/script.sh' first and rerun command."
```